modules/rhcos-enable-node-scan.adoc

```diff
@@ -9,7 +9,7 @@
 If you use {ocp}, you can enable scanning of Red Hat Enterprise Linux CoreOS (RHCOS) nodes for vulnerabilities by using {rh-rhacs-first}.
 
 .Prerequisites
-* You must use {ocp} 4.10 or later for scanning RHCOS node hosts.
+* For scanning RHCOS node hosts of the Secured cluster, you must have installed Secured cluster on {ocp} 4.10 or later. For more details on supported managed and self-managed {ocp} versions, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```

Comment:
I do not have access to your common attributes file. Should the first instance of Red Hat Enterprise Linux CoreOS an subsequent instances of RHCOS be variables?